src/irmin-pack/inode_intf.ml

```diff
@@ -83,6 +83,24 @@ module type INTER = sig
 
     val integrity_check : t -> bool
     (** Checks the integrity of an inode. *)
+
+    module Tree : sig
```

Comment:
What do you think about `Contrete` instead of `Tree`? This was we can mimic the `to_concrete` and `of_concrete` functions of `tree.ml`

Reply:
good idea! I'll modify this.